### DISCUSSION TITLE
Problem with the generate hill wallpaper that after wake up from sleep produces the slider at the bottom and left of the screen

### DISCUSSION THREAD

**Discussion by Blank-labs** [rocksdanister/lively#2409] [Q&A]
![image_2024-09-28_194634786](https://github.com/user-attachments/assets/6b491414-9a81-4107-b263-8bc7a8c4434e)

**Comment by Blank-labs** [comment]
to fix it i have to reapply the wallpaper but i want to know how to prevent it

**Reply by rocksdanister** [reply]
You can add this style to the wallpaper file
https://www.w3schools.com/howto/howto_css_hide_scrollbars.asp
